src/pages/docs/projects/project-triggers/git-triggers.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-11-05
+modDate: 2024-11-05
+title: Git repository triggers
+description: Git repository triggers allow you to automatically create a new release when a new commit is pushed to a Git repository.
```

Comment:
As we had chatted about separately, it might be good to have some sort of introduction to this page, this description reads pretty nicely so perhaps we can go with this for the moment?

Reply:
Added